apps/wing-console/console/ui/src/layout/discord-button.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTheme } from "@wingconsole/design-system";
+import classNames from "classnames";
+
+import { DiscordIcon } from "./discord-icon.js";
+
+const WING_DISCORD_URL = "https://discord.gg/FBvMBQQkqR";
```

Comment:
@skyrpex it is not the correct URL. Please use `https://t.winglang.io/discord`

Reply:
Maybe add a utm_source or something that will allow us to track where the join came from?